about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/d404a68fe3.js" crossorigin="anonymous"></script>
+    <script src="script.js" defer></script>
+    <title>About section</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" class="check-btn">
+            <i class="fa-solid fa-bars fa-2x" id="bars-icon" onclick="myFunction(this)"></i>
+        </a>
+        <nav class="navbarNav">
+            <a href="index.html" class="logo"><img src="assets/saldana-degisn.png" alt="logo"></a>
+            <div class="nav-options">
+                <ul class="nav-list">
+                    <li class="nav-items"><a href="about.html">About</a></li>
```

Comment:
- [ ] Kindly change this to index.html so the user can easily navigate to the home page from the about 👍🏿